apps/hash-frontend/src/shared/icons/upload-icon.tsx

```diff
@@ -0,0 +1,16 @@
+import { SvgIcon, SvgIconProps } from "@mui/material";
+import { FunctionComponent } from "react";
+
+export const UploadIcon: FunctionComponent<SvgIconProps> = (props) => {
+  return (
+    <SvgIcon
+      {...props}
+      width="512"
+      height="512"
+      viewBox="0 0 512 512"
+      fill="none"
+    >
```

Comment:
We may want these to be overridable
```suggestion
    <SvgIcon
      width="512"
      height="512"
      viewBox="0 0 512 512"
      fill="none"
      {...props}
    >
```

Reply:
Fixed in https://github.com/hashintel/hash/pull/3104/commits/85bdc36be999274dcf0ba041366d37b00b1df180